src/BBT.Maybe/NoneCase.cs

```diff
@@ -7,83 +7,83 @@
     /// </summary>
     public struct NoneCase : IEquatable<NoneCase>
     {
-        private readonly bool mIsNone;
+        private readonly bool isNone;
 
         /// <summary>
         /// Initializes a new instance of the <see cref="NoneCase"/> struct.
         /// </summary>
-        /// <param name="aIsNone">True if representing the none case, false if value is set.</param>
-        internal NoneCase(bool aIsNone)
+        /// <param name="isNone">True if representing the none case, false if value is set.</param>
+        internal NoneCase(bool isNone)
         {
-            this.mIsNone = aIsNone;
+            this.isNone = isNone;
         }
 
         /// <summary>
         /// Checks whether the operands are equal.
         /// </summary>
-        /// <param name="aA">Maybe to compare.</param>
-        /// <param name="aB">Maybe to compare.</param>
-        public static bool operator ==(NoneCase aA, NoneCase aB)
+        /// <param name="a">Maybe to compare.</param>
+        /// <param name="b">Maybe to compare.</param>
+        public static bool operator ==(NoneCase a, NoneCase b)
         {
-            return aA.mIsNone.Equals(aB.mIsNone);
+            return a.isNone.Equals(b.isNone);
         }
 
         /// <summary>
         /// Checks whether the operands are unequal.
         /// </summary>
-        /// <param name="aA">Maybe to compare.</param>
-        /// <param name="aB">Maybe to compare.</param>
-        public static bool operator !=(NoneCase aA, NoneCase aB)
+        /// <param name="a">Maybe to compare.</param>
+        /// <param name="b">Maybe to compare.</param>
+        public static bool operator !=(NoneCase a, NoneCase b)
         {
-            return !(aA == aB);
+            return !(a == b);
```

Comment:
`return !(a == b);` may be simplified by `return a != b;`

Reply:
Done.